hal/src/boron/network/sara_ncp_client.cpp

```diff
@@ -494,6 +500,13 @@ int SaraNcpClient::queryAndParseAtCops(CellularSignalQuality* qual) {
     cgi_.mobile_country_code = static_cast<uint16_t>(::atoi(mobileCountryCode));
     cgi_.mobile_network_code = static_cast<uint16_t>(::atoi(mobileNetworkCode));
 
+    if (ncpId() == PLATFORM_NCP_SARA_R410) {
+        if (act == static_cast<int>(CellularAccessTechnology::LTE)) {
```

Comment:
(Just FYI) There is also `particle::to_underlying` in `enumclass.h` to make the enum class conversions a bit more pretty.